optimum/bettertransformer/models/__init__.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
+
+from . import albert, bart, bert, distilbert
+
+
+BETTER_TRANFORMER_LAYERS_MAPPING_DICT = {
+    # Bert Family
+    "BertLayer": bert.BertLayerBetterTransformer,
+    "ElectraLayer": bert.BertLayerBetterTransformer,
+    "Data2VecTextLayer": bert.BertLayerBetterTransformer,
+    "CamembertLayer": bert.BertLayerBetterTransformer,
+    "MarkupLMLayer": bert.BertLayerBetterTransformer,
+    "RobertaLayer": bert.BertLayerBetterTransformer,
+    "SplinterLayer": bert.BertLayerBetterTransformer,
+    "ErnieLayer": bert.BertLayerBetterTransformer,
+    "LayoutLMLayer": bert.BertLayerBetterTransformer,
+    "BertGenerationLayer": bert.BertLayerBetterTransformer,
+    "RobertaLayer": bert.BertLayerBetterTransformer,
+    "Data2VecTextLayer": bert.BertLayerBetterTransformer,
+    "XLMRobertaLayer": bert.BertLayerBetterTransformer,
+    # Albert Family
+    "AlbertLayer": albert.AlbertLayerBetterTransformer,
+    # Bart family - need to tweak the tests a bit
+    "BartEncoderLayer": bart.BartLayerBetterTransformer,
+    # "PLBartEncoderLayer": bart.BartLayerBetterTransformer,
+    # "MarianEncoderLayer": bart.BartLayerBetterTransformer,
+    # "TimeSeriesTransformerEncoderLayer": bart.BartLayerBetterTransformer,
+    # "BlenderbotSmallEncoderLayer": bart.BartLayerBetterTransformer,
+    # T5 family - needs to check compatibility first
+    # "T5Block": t5.T5LayerBetterTransformer,
+    # Some models cannot be tested such as:
+    # "QDQBertLayer": bert.BertLayerBetterTransformer, --> needs torch quantization
+    # "RealmLayer": bert.BertLayerBetterTransformer, --> not mapped in AutoModel
+    "TransformerBlock": distilbert.DistilBertLayerBetterTransformer,
+}
+
+
+def is_module_fast(module_name):
```

Comment:
I wouldn't keep this method, it is used only once, and the logic is simple. But it's taste. In any case, I think having control flow with different return type should be avoided.

Reply:
Looks OK to me

Reply:
Nice! Thanks a lot 💪